packages/vue/src/components/sign-in.vue

```diff
@@ -118,12 +118,13 @@ const onForgotPasswordClicked = (): void => {
           <base-field-set
             :disabled="actorState.matches('signIn.submit')"
             class="amplify-flex"
+            p
             style="flex-direction: column"
           >
             <template #fieldSetI="{ slotData }">
               <slot name="signin-fields" :info="slotData"> </slot>
             </template>
-
+            <legend class="amplify-visually-hidden">Sign in</legend>
```

Comment:
Please post a screenshot of the Sign In Vue, Angular, and React Authenticator Sign In screens. We should also make this accessibility change to all the other screens with forms as well, as they will have the same problem.

Reply:
Screenshots updated in the description. We can cut a ticket for other forms since the [current ticket](https://app.asana.com/0/1201736086077838/1201759796183705/f) is only about fixing a11y for homepage 😁

Reply:
Good point, sorry for the scope creep 😭